templates/spring-kotlin/files/.environment

```diff
@@ -0,0 +1,2 @@
+export JAVA_MEMORY=-Xmx$(jq .info.limits.memory /run/config.json)m
+export JAVA_OPTS="$JAVA_MEMORY -XX:+ExitOnOutOfMemoryError"
```

Comment:
Same question for these, I guess.  If you don't want to inline them in the command, why not a multi-line start?